src/Tribe/Views/V2/Widgets/Widget_List.php

```diff
@@ -65,41 +70,94 @@ class Widget_List extends Widget_Abstract {
+
+		$default_arguments['description'] = esc_html_x( 'A widget that displays upcoming events.', 'The description of the List Widget.', 'the-events-calendar' );
+		// @todo update name once this widget is ready to replace the existing list widget.
+		$default_arguments['name']                          = esc_html_x( 'Events List V2', 'The name of the widget.', 'the-events-calendar' );
+		$default_arguments['widget_options']['description'] = esc_html_x( 'A widget that displays upcoming events.', 'The description of the List Widget.', 'the-events-calendar' );
+		// Setup default title.
+		$default_arguments['title'] = _x( 'Upcoming Events', 'The default title of the List Widget.', 'the-events-calendar' );
+
+		return $default_arguments;
+	}
 
 	/**
-	 * @todo update in TEC-3612 & TEC-3613
-	 *
 	 * {@inheritDoc}
 	 */
-	public function setup() {
-		$this->setup_view();
+	public function update( $new_instance, $old_instance ) {
+		$instance = $old_instance;
+
+		/* Strip tags (if needed) and update the widget settings. */
+		$instance['title']                = wp_strip_all_tags( $new_instance['title'] );
+		$instance['limit']                = $new_instance['limit'];
+		$instance['no_upcoming_events']   = ! empty( $new_instance['no_upcoming_events'] );
+		$instance['featured_events_only'] = ! empty( $new_instance['featured_events_only'] );
+		$instance['jsonld_enable']        = (int) ( ! empty( $new_instance['jsonld_enable'] ) );
+
+		return $this->filter_updated_instance( $instance );
 	}
 
 	/**
 	 * {@inheritDoc}
 	 */
-	public function get_arguments() {
-		$arguments = $this->arguments;
+	public function get_admin_fields() {
 
-		$arguments['description'] = esc_html__( 'A widget that displays upcoming events.', 'the-events-calendar' );
-		// @todo update name once this widget is ready to replace the existing list widget.
-		$arguments['name']                          = esc_html__( 'Events List V2', 'the-events-calendar' );
-		$arguments['widget_options']['description'] = esc_html__( 'A widget that displays upcoming events.', 'the-events-calendar' );
+		return [
+			'title'                => [
+				'label' => _x( 'Title:', 'The label for the field of the title of the List Widget.','the-events-calendar' ),
+				'type'  => 'text',
+			],
+			'limit'                => [
+				'label'   => _x( 'Show:', 'The label for the amount of events to show in the List Widget.', 'the-events-calendar' ),
+				'type'    => 'dropdown',
+				'options' => $this->get_limit_options(),
+			],
+			'no_upcoming_events'   => [
+				'label' => _x( 'Show widget only if there are upcoming events', 'The label for the option to hide the List Widget if no upcoming events.', 'the-events-calendar' ),
+				'type'  => 'checkbox',
+			],
+			'featured_events_only' => [
+				'label' => _x( 'Limit to featured events only', 'The label for the option to only show featured events in the List Widget.', 'events list widget setting', 'the-events-calendar' ),
```

Comment:
:no_entry_sign: Too many arguments for function "_x".

<sub>hash: e39f39df71163231eff8dff38ba1b73c</sub>